test/services/CashTransactionsServiceSpec.scala

```diff
@@ -181,6 +185,90 @@ class CashTransactionsServiceSpec extends SpecBase {
       }
     }
   }
+  "submitCashAccountStatementRequest" should {
+
+    "return Left with ErrorDetail if api request failed" in new Setup {
+
+      val casErrorDetailStr01 =
+        """
+          |{
+          |  "timestamp": "2024-01-21T11:30:47Z",
+          |  "correlationId": "f058ebd6-02f7-4d3f-942e-904344e8cde5",
+          |  "errorCode": "400",
+          |  "errorMessage": "Request could not be processed",
+          |  "source": "Backend",
+          |  "sourceFaultDetail": {
+          |    "detail": [
+          |      "Invalid JSON message content used"
+          |    ]
+          |  }
+          |}""".stripMargin
+
+      val casErrorResponseDetails01: ErrorDetail = Json.fromJson[ErrorDetail](
+        Json.parse(casErrorDetailStr01)).get
+
+      when(mockAcc45Connector.submitStatementRequest(cashAccSttReqDetail)).thenReturn(
+        Future.successful(Left(casErrorResponseDetails01))
+      )
+
+      running(app) {
+        val result = await(service.submitCashAccountStatementRequest(cashAccSttReqDetail))
+        result mustBe Left(casErrorResponseDetails01)
+      }
+    }
+
+    "return Right with ResponseCommon if api request successful" in new Setup {
```

Comment:
Better to say as below

"return Right with ResponseCommon if api request is successful"